pkg/test/controller.go

```diff
@@ -114,6 +118,13 @@ func seedTestData(t *testing.T, ctx context.Context, d Data) (Clients, Informers
 			t.Fatal(err)
 		}
 	}
+	c.Pipeline.PrependReactor("*", "customruns", test.AddToInformer(t, i.CustomRun.Informer().GetIndexer()))
```

Comment:
I would suggest not to use this and this is temporary and removed later, this is very specific for v1beta1 of resources which are deprecated

Reply:
Thank you for your suggestion. Since this is my first PR, I referred the implementation of TaskRun. If it is not recommended to use `c.Pipeline.PrependReactor`, then which method should be used as a replacement?

Reply:
If this temporary implementation will be replaced later, can we keep and wait for it to be replaced in subsequent PR?

Reply:
I am not saying about c.Pipeline.PrependReactor. I am saying that this whole function `seedTestData` will be removed. 

You have to use `SeedTestData` in tests instead of `SeedV1beta1TestData` and you will not need these changes in test pkg. This test pkg is very specific for resources whose v1 is available and v1beta1 is deprecated like task, taskrun, pipeline and pipelinerun